src/drivers/frsky_telemetry/frsky_data.c

```diff
@@ -266,7 +266,7 @@ void frsky_send_frame2(int uart)
 		lon    = frsky_format_gps(fabsf(global_pos->lon));
 		lon_ew = (global_pos->lon < 0) ? 'W' : 'E';
 		speed  = sqrtf(global_pos->vel_n * global_pos->vel_n + global_pos->vel_e * global_pos->vel_e)
-			 * 25.0f / 46.0f;
+			 * 1.94384; // knots per m/s (was 25.0f / 46.0f = 0.543)
```

Comment:
Make it a float literal. `1.94384f`

Reply:
What was the previous calculation supposed to be?

Reply:
Yes I have now. Do I close this pull request and make another?

Reply:
I don't know, it had always been low by a factor of almost 3.6, almost as if the PixHawk source speed value was in km/hour instead of m/s.

Reply:
Oh well, at least it's right now. No need to keep the comment about what it was before.

Reply:
I've fixed the comment.